teachopencadd/talktorials/T032_compound_activity_proteochemometrics/talktorial.ipynb

```diff
@@ -0,0 +1,1920 @@
+{
```

Comment:
<p>Same question here:</p><p>Instead of</p><pre class="ql-syntax" spellcheck="false">from mordred import Calculator, descriptors
from prodec import ProteinDescriptors
</pre><p>can we do</p><pre class="ql-syntax" spellcheck="false">import mordred
import prodec
</pre><p>and use</p><pre class="ql-syntax" spellcheck="false">mordred.Calculator
mordred.descriptors
prodec.ProteinDescriptors
</pre><p>?</p><p> </p><p>And - in TeachOpenCADD, we use f-strings</p><pre class="ql-syntax" spellcheck="false">print('Available ProDEC descriptors: ', desc_factory.available_descriptors)
</pre><p>would become</p><pre class="ql-syntax" spellcheck="false">print(f"Available ProDEC descriptors: {desc_factory.available_descriptors}")
</pre><p>In this case, the complexity is not reduced by much but especially when you join together more text and variable bits, f-strings are my best friend.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>

Reply:
<p>I did this as much as possible. Where the imports would not work I created a nickname (i.e. mordred.descriptors &gt; mordred_descriptors). </p><p> </p><p>Good catch! I used f-strings all over the notebook, but at some points consistency left me. </p>